src/test/java/com/teragrep/pth10/steps/teragrep/bloomfilter/TeragrepBloomFilterTest.java

```diff
@@ -145,7 +145,14 @@ void testSavingToDatabase() {
         byte[] filterBytes = (byte[]) row.get(1);
         BloomFilter rawFilter = Assertions
                 .assertDoesNotThrow(() -> BloomFilter.readFrom(new ByteArrayInputStream(filterBytes)));
-        TeragrepBloomFilter filter = new TeragrepBloomFilter(partition, rawFilter, lazyConnection.get(), filterTypes);
+        TeragrepBloomFilter filter = new TeragrepBloomFilter(
```

Comment:
Not this row, but in this File there are the dpl.pth_06.bloom.pattern and dpl.pth_06.bloom.pattern put into the properties in the setEnv() function. Is that necessary anymore?

Reply:
Removed since not needed anymore